packages/react-components/react-combobox/docs/Spec.md

```diff
@@ -22,20 +24,23 @@ Select provides better mobile support and accessibility than Combobox, and has t
 
 Use Select when a basic single-select form component with no freeform text input or filtering is needed.
 
-### When to use Combobox
+### When to use Combobox or Dropdown
```

Comment:
Consider breaking this into two sections, one for Combobox and one for Dropdown. Above you state that the controls discussed in this section cannot be used interchangeably but then these two controls are grouped together implying interchangeability.

Reply:
I was aiming to break them up by package, but I definitely see how it was confusing. I've updated the headings and wording a bit -- let me know if that reads OK, or if you think I should still just merge the two sections together.